keystone_spassword/contrib/spassword/core.py

```diff
@@ -159,32 +190,32 @@ def update_user_modification_time(self, user):
         raise exception.NotImplemented()
 
 
-class SPasswordMiddleware(wsgi.Middleware):
+# class SPasswordMiddleware(wsgi.Middleware):
 
-    def __init__(self, *args, **kwargs):
-        LOG.debug("SPasswordMiddleware INIT")
-        try:
-            self.spassword_api = SPasswordManager()
-        except Exception:
-            LOG.debug("SPasswordMiddleware already registered")
-        return super(SPasswordMiddleware, self).__init__(*args, **kwargs)
+#     def __init__(self, *args, **kwargs):
```

Comment:
Leftover comment block?

Reply:
Fixed in: https://github.com/telefonicaid/fiware-keystone-spassword/pull/57/commits/09846cb1bead252b70340d4a8f033c750aa0cd3f